packages/contracts/test/LibExperience.t.sol

```diff
@@ -24,7 +25,50 @@ contract LibExperienceTest is MudV2Test {
 
     uint32[PStat_length] memory exp = LibExperience.getExp(targetEntity);
     for (uint256 i; i < exp.length; i++) {
-      assertEq(exp[i], 1);
+        assertEq(exp[i], 1);
     }
   }
+
+  function testHasExp() public {
+    assertFalse(LibExperience.hasExp(targetEntity));
```

Comment:
why change `testHasExp`? it was the only test I liked from the start

Reply:
I'm not sure here, before you indicated to delete the extra initialization lines
```
    LibExperience.initExp(targetEntity);
    assertTrue(LibExperience.hasExp(targetEntity));
```

Reply:
> you can remove
> ```solidity
> LibExperience.initExp(targetEntity);
> assertTrue(LibExperience.hasExp(targetEntity));
> ```
> from all other tests, since you add a dedicated existence check

I meant that `testHasExp` does an existence check, so other tests don't need it